### ISSUE TITLE
Remove routing mark

### ISSUE THREAD

**Issue by sindresorhus** [tastejs/todomvc#812]
This one is also not very useful. We should instead strive to get all the apps to implement routing.

Blocked by `Let's also check if we want to keep all of the current examples.` in https://github.com/tastejs/todomvc/issues/735

**Comment by arthurvr** [comment]
> Agreed - the majority do implement routing, so I would push to have all implement it.

Shall I ping the authors of all apps that don't have routing yet in this issue? It's getting time to implement it :)

**Comment by sindresorhus** [comment]
Yes.

**Comment by addyosmani** [comment]
Yes please!

**Comment by arthurvr** [comment]
Hey folks! Could y'all checkout this issue? We'd love to have routing integrated in all apps. You can use the existing apps as example. Be sure to read the [app spec](https://github.com/tastejs/todomvc/blob/master/app-spec.md#routing) as well.

* [ ] angular-dart: @passy 
* [ ] typescript-backbone: I was unable to figure out who built this one. Was it @addyosmani?
* [ ] cujo: @briancavalier
* [ ] olives: @podefr
* [ ] duel: @mckamey
* [ ] dijon: @creynders

Thanks!

**Comment by podefr** [comment]
hey, thanks for the ping. I'll add routing to Olives and will create a pull request.

**Comment by arthurvr** [comment]
Great, thanks!

**Comment by mckamey** [comment]
@arthurvr I am interested in updating for DUEL. Life has been busy but will check it out soon.

**Comment by arthurvr** [comment]
Hey @podefr, is there still interest in picking this up? Thanks!

**Comment by podefr** [comment]
Yes @arthurvr, I'll take care of this

**Comment by arthurvr** [comment]
ping @podefr :)

**Comment by podefr** [comment]
@arthurvr I can't give you an ETA, when do you need this for? It may take a couple of months. The problem isn't the router as I've got this working somewhere in a new branch, but I need to implement a new feature to be able to filter a model without breaking data-binding, and this may take a while due to priorities.

**Comment by arthurvr** [comment]
@podefr No problem I guess :) Keep us updated.

**Comment by arthurvr** [comment]
@mckamey still interested?

**Comment by mckamey** [comment]
@arthurvr, yes still interested. I have a newborn at home so life is quite busy.

**Comment by arthurvr** [comment]
No worries @mckamey. Congrats!

**Comment by arthurvr** [comment]
ping @mckamey

**Comment by mckamey** [comment]
Had to update the DUEL example to use latest template first which involved reorienting myself with all of this. Thinking I'll use the suggested [Flatiron Director](https://github.com/flatiron/director) to assist with routing.

**Comment by arthurvr** [comment]
This issue was closed by accident. PRs still welcome :)

**Comment by addyosmani** [comment]
Thanks so much for this contribution. I'm going through old issues and closing them out as we last updated the project in a major way in 2024 and want to ensure we're focused on only those issues still relevant. Thanks once again, feel free to reopen if still relevant
